docs/contrib/quickstart.rst

```diff
@@ -100,19 +100,28 @@ Then you'll be able to use locally published version in other projects.
 Locally build docs
 --------------------
 
-1. First time building the docs. This command will setup & build the docs.
+1. With the current directory set to the project root, execute:
 
 .. code-block:: text
 
-    $ bash scripts/makedocs setup
+    $ bash scripts/makedocs 
 
-2. If setup is already done. This command will only build the docs assuming setup is already done.
+2. For use in the next step, note the full, absolute path to the
+   ``docs/_build/html`` directory. For example:
 
 .. code-block:: text
+		
+    /Users/Νίκη/MyScalaNativeProject/docs/_build/html
+    
+3. Verify the results by opening the ``index.html`` of that directory
+   in your favorite browser using a URL of the form
+   ``file://<absolute_path>/index.html``,
+   where ``<absolute_path>`` is the path
+   noted in step 2. For example:
 
-    $ bash scripts/makedocs 
+.. code-block:: text
 
-3. Navigate to ``docs/_build/html`` directory and open ``index.html`` file in your browser.
+    file:///Users/Νίκη/MyScalaNativeProject/docs/_build/html/index.html
```

Comment:
Maybe better keep using paths relative to project root directory

Reply:
Thank you for looking at this.

In general, I concur that things like `scripts/makedocs` should, after the first mention of 
`from project root directory`, be assumed and documented as relative

To the best of my knowledge a `file:` URL scheme has to be a fully qualified path.  I think having a
working example makes it easier for devos who do not use `file` URLs regularly to check their work.